docker-bake.hcl

```diff
@@ -14,7 +14,7 @@ variable "DOCKER_BUILDTAGS" {
   default = ""
 }
 variable "DOCKER_GITCOMMIT" {
-  default = "HEAD"
+  default = ""
```

Comment:
I think @crazy-max mentioned there's a `null` option now as well, but perhaps he could have a look how to use these?

Reply:
Yes if you set to `null` then you can keep `ARG DOCKER_GITCOMMIT=HEAD`. That makes sense to me.

Reply:
This way if `DOCKER_GITCOMMIT` is not set it will keep `HEAD` instead of setting it to an empty string in `ARG`.

Reply:
Done, thanks.